packages/vsphere-cpi/package.yaml

```diff
@@ -0,0 +1,3 @@
+url: local
+packageVersion: 00
+releaseCandidateVersion: 00
```

Comment:
Technically this falls to the whole 1.0.000 semver problem, but I think its ok functionality wise.

Reply:
The semver problem here was that 1.0.000 would be interpreted as 1.0.0 right? I think that actually works to our favor here since, for local charts, Rancher shouldn't be using packageVersion at all (since we should just bump the patch version on each of our changes)

Reply:
That or the compare would error out. I haven't seen that actually fail though.